File: app/src/main/java/moises/com/usersapp/ui/UsersApp.java
package moises.com.usersapp.ui;

import android.app.Application;
import android.content.Context;

public class UsersApp extends Application{

    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = getApplicationContext();
    }

    public static Context getContext() {
        return mContext;
    }
}

File: app/src/main/java/moises/com/usersapp/ui/UsersApp.kt
package moises.com.usersapp.ui

import android.app.Application
import dagger.hilt.android.HiltAndroidApp
import timber.log.Timber

@HiltAndroidApp
class UsersApp : Application() {

    override fun onCreate() {
        super.onCreate()
        Timber.plant(Timber.DebugTree())
    }
}
